packages/react/src/styles.css

```diff
@@ -0,0 +1,45 @@
+/* inputs */
+
+[data-sr-input] {
+  width: 100%;
+  color: black;
+}
+
+[data-sr-input-label] {
+  padding-bottom: 0.2rem;
+  display: block;
+}
+
+[data-sr-input-asterisk] {
+  color: hotpink;
+}
+
+/* buttons */
+
+[data-sr-button] {
+  display: flex;
+  gap: 0.5rem;
+}
+
+[data-sr-button='primary'] {
+  padding: 0.5rem 1rem;
+  background: hotpink;
+  color: white;
+}
+
+[data-sr-button='secondary'] {
+  padding: 0.5rem 1rem;
+  border: 2px solid hotpink;
+  color: hotpink;
+}
+
+[data-sr-button='tertiary'] {
+}
+
+[data-sr-button][disabled] {
+  opacity: 0.8;
+  cursor: wait;
+}
```

Comment:
we can treat these individually per product AND/OR based on context/kind.
for example, we could have this to brand a convertkit subscribe form:
```css
[data-sr-convertkit-subscribe-form='landing'] {
  @apply space-y-6 w-full max-w-md mx-auto flex flex-col;
  [data-sr-input] {
    @apply w-full bg-gray-900 sm:text-xl px-5 py-4 rounded-lg text-white placeholder-gray-300 border-gray-700 shadow-inner focus:ring-0 focus:border-orange-300;
  }
  [data-sr-input-label] {
    @apply sm:text-lg text-base block pb-2;
    [data-sr-input-asterisk] {
      @apply text-orange-300;
    }
  }
  [data-sr-button='primary'] {
    @apply sm:px-20 px-16 py-4 text-lg font-semibold self-center bg-orange-400 bg-opacity-90 hover:bg-opacity-100 border-2 border-orange-400 text-white rounded-lg transform hover:shadow-xl hover:scale-105 focus:scale-95 transition-all ease-in-out duration-200 focus:outline-none outline-none focus:ring-2 focus:ring-orange-200;
  }
  [data-sr-button='primary'][disabled] {
    @apply opacity-80 cursor-wait hover:scale-100;
  }
}

```